### DISCUSSION TITLE
Inquiry mode don't work Gamecube

### DISCUSSION THREAD

**Discussion by vezinito** [darthcloud/BlueRetro#618] [Q&A] [answered]
Hello friends i use a ESP32 internal on my Gamecube with 1.8.1 and now too 1.8.3v hw2 version, and I'm not capable to pair any controller, the LED is always static ( tried to push boot and EN buttons so many times in different sequences, no luck )

https://ibb.co/Br1T7Wd
https://ibb.co/k2jywyQ

-I check the physical install so many times, seems ok.
-I change the cr2032 battery for new one.
-With blueretro.io and mobile phone connection work perfect and detect 1.8.3v hw2 version blueretro OK.

Thanks in advance.

**EDIT: Fixed, the problem was using HW2 version for ESP32 module, is required to use HW1 firmware, now work fine.**

**Comment by darthcloud** [comment, marked as answer]
Use HW1 firmware when you follow HW1 wiring
